docs/config.rst

```diff
@@ -102,6 +102,8 @@ examples first and use this page as a reference.
 Core
 ----
 
+The following options are set in the ``[tox]`` section of ``tox.ini`` or the ``[tox:tox]`` section of ``setup.cfg``.
```

Comment:
For what's worth, part of the reason we did not document this here is that people can add new configuration formats via plugins, or via `toxfile.py` when these don't exists but the settings are still valid.

Reply:
Ok, that's good to know. I think it's still worth more clearly documenting the standard mechanisms, even if we don't cover the alternative mechanisms yet, but I will prepare another PR to work on that issue.

Reply:
As a starting point for discussion, I've put an initial go at further changes related to this on my dev/cjs/24a20/doc-config-mechanisms branch. I am far from sure I understand yet what you're saying about configuration mechanisms, but this is what I've got so far.